internal/command/checks/input.go

```diff
@@ -0,0 +1,57 @@
+package checks
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/superfly/flyctl/api"
+)
+
+func isInterrupt(err error) bool {
+	return err != nil && err.Error() == "interrupt"
+}
+
+func selectOrganization(ctx context.Context, client *api.Client, slug string) (*api.Organization, error) {
```

Comment:
There is a reusable organization selector in the prompt package: https://github.com/superfly/flyctl/blob/832c567731a1ea687a83af76191ff20e767eb23e/internal/prompt/prompt.go#L183

Reply:
Also addressed in 2baef524. thanks!